index.html

```diff
@@ -427,9 +428,93 @@ <h1>参加者の声</h1>
         </div>
       </div>
     </section>
-    <!-- section-team-position -->
+    <!-- section-participant-feedback-position -->
+
+    <section id="section-faq-position" class="section-wrapper section-faq-position">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-12">
+            <div class="section-title">
+              <h1>よくある質問</h1>
+            </div>
+            <div class="section-message">
+              <p>初めて参加される方からよくいただいている質問にお答えします。</p>
+            </div>
+          </div>
+        </div>
+        <div class="section-faq">
+          <ol>
+            <li>
+              <p class="section-faq-question">ハッカソン未経験だと参加は難しいですか？</p>
+              <p class="section-faq-answer">A. ハッカソン未経験の方も大歓迎です！</p>
+              <span>
+                周りの参加者と助け合ったり、メンターのサポートを受けることができます。
+                Takizawa Hackathonはものづくりに興味のある学生間の交流の場を作るために立ち上がりました。
+                集まった仲間とチームを立ち上げ、目的を共有し、サービスをゼロから作るイベントです。
+                ハッカソン未経験で参加を悩まれている方にも、ぜひこの機会にエントリーいただければと思います！
+              </span>
+            </li>
+            <li>
+              <p>開発経験がないのですが、参加したら迷惑になりませんか？</p>
+              <p class="section-faq-answer">A. 開発未経験者でも全く問題ありません！</p>
+              <span>
+                個人でエントリーする場合でもエントリー時に提出してもらった技術力アンケートを基にチーム分けを行います。また、社会人メンターとして現役エンジニアが各チームをサポートします！
+              </span>
+            </li>
+            <li>
+              <p>サービス開発が初めてのため企画づくりが不安です。どうすれば良いでしょうか？</p>
+              <p class="section-faq-answer">A. 企画づくりからサポートします！</p>
+              <span>
+                ハッカソンでは、「何を作れば良いだろうか」とアイデア発想や企画づくりの段階で時間がかかってしまうことがあります。
+                今回の Takizawa Hackathon では、スポンサー企業のメンターによる「実際の現場で行われている企画の進め方」をレクチャーする時間を設ける予定です。
```

Comment:
こちらまだ確定ではないと思うのでもうちょっと抽象的な表現の方が良いかなと思いましたがいかがでしょう！

Reply:
こちらでもちょっと考えてみていました:bow:
https://takizawahackathon.slack.com/archives/C5G6A577C/p1700280187073049?thread_ts=1700240285.798159&cid=C5G6A577C
> Takizawa Hackathonでは、企画が不安な方にも安心して参加できるように、アイディア出しの段階で悩んでいるチームにはメンターが相談に乗ったり、アドバイスを受けたりすることができます。

Reply:
@sasakitomohiro 
確かにまだ未確的な部分だと思うので、もう少しふわっとした表現の方が良さそうですね〜
以下のような感じでどうでしょうか？

```suggestion
                今回の Takizawa Hackathon では、イベントの最初にアイディアの発想、収束の方法を体験していただけるプログラムを用意させていただく予定です。
```

Reply:
ご確認ありがとうございます！
良さそうだと思いました！
考えていただきありがとうございます:bow:

Reply:
あ、すみません。Slackの方でのやり取り見逃してました！
https://takizawahackathon.slack.com/archives/C5G6A577C/p1700280187073049?thread_ts=1700240285.798159&cid=C5G6A577C

ともひろ君の案のが良さそうです！

```suggestion
                Takizawa Hackathonでは、企画が不安な方にも安心して参加できるように、アイディア出しの段階で悩んでいるチームにはメンターが相談に乗ったり、アドバイスを受けたりすることができます。
安心してご参加ください。
```

Reply:
ご確認ありがとうございます！
そしたらこちらでいきますか！:pray::bow::pary:

Reply:
3ab17ef 反映しました！

Reply:
ありがとうございます！